src/server.js

```diff
@@ -53,6 +53,7 @@ Translators.init()
 	if (process.env.NODE_ENV == 'test') return;
 	
 	var port = config.get('port');
-	app.listen(port);
-	Debug.log(`Listening on 0.0.0.0:${port}`);
+	var host = config.has('host') ? config.get('host') : '0.0.0.0';
```

Comment:
Since this will be in default.json, we can just make it `config.get('host')`, as with `port`.